docs/README.md

```diff
@@ -1,56 +1,102 @@
 # What is cmdalias
 
-`cmdalias` is a tool that I created for myself because I was tired of using too many keystrokes for commands that i'm using everyday in my shell. For example I often want to list images from my docker.
+`cmdalias` is an alias tool inspired from git-config that I created for myself. Moreover, it lets you create
```

Comment:
I think you have an auto linewrap on your editor :-) This should all be on the same line,

Reply:
Will fix :)